fix(toggle-switch): apply lower opacity when disabled

dancormier opened the pull request:
This PR applies the disabled element opacity to `s-toggle-switch` components. Previously, the disabled state for `input[type="checkbox].s-toggle-switch` would result in no visual changes.

### Before

![image](https://github.com/StackExchange/Stacks/assets/647177/b24efb65-799b-451b-a141-1b9abfa2f78c)

### After

![image](https://github.com/StackExchange/Stacks/assets/647177/65ef3a94-0f34-482c-b279-d151866cb1c0)


---

> **Note**
> This should be good to merge after we containerize visual regression tests

dancormier committed:
fix(toggle-switch): apply lower opacity when disabled

dancormier committed:
add visual and a11y tests

dancormier committed:
lint!

giamir commented:
@dancormier out of curiosity why we changed so many unrelated baseline images as part of this PR?

dancormier committed:
Revert "lint!"

This reverts commit 28a56b575adce24912f0428eebae73a7f7df395c.

dancormier committed:
Revert "add visual and a11y tests"

This reverts commit b467f723d494bc8a75c93cc34a676ce1a1e6f3ab.

dancormier committed:
Add visual and a11y tests (again)

dancormier commented:
> @dancormier out of curiosity why we changed so many unrelated baseline images as part of this PR?

I'm not sure.Looks like when I run `npm run test:visual:update` it makes _minor_ changes to every image.

![image](https://github.com/StackExchange/Stacks/assets/647177/84d17f69-3379-4418-a130-100c364bd355)

I'm gonna dig into commits between the last time we've updated the images and now to see if there's any change that might have caused this..

dancormier commented:
I went back to the last time baseline images were updated (https://github.com/StackExchange/Stacks/commit/a813110cedb915d9b80d985c17dc87d4fa8a5153), tried another update, and saw the same thrashing. My initial theory is that an OS or browser update has modified how these components get rendered, but that theory might not explain why tests are failing via GitHub actions.

I'm going to open a new PR that includes regenerated baseline images and see if it resolves test failures and consider merging in those updated images. Expect a PR review request @giamir

Edit: I just noticed they're all Chromium images that are getting regenerated, so I'm guessing a browser update included a slight rendering update 🤷‍♂️

dancormier committed:
Merge branch 'develop' into dcormier/toggle-switch-disabled-fix

dancormier committed:
Add baseline images for toggle switch

dancormier committed:
Trying to make the tests pass…

dancormier committed:
Merge branch 'develop' into dcormier/toggle-switch-disabled-fix
